office/ProjectLibre/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           java 1.0
+
+github.setup        giansalvo ProjectLibre 1.8.0 v
```

Comment:
from the [project website](http://www.projectlibre.com/) the link to the community edition points to [SourceForge](https://sourceforge.net/projects/projectlibre/). Is there a reason you use your own GitHub fork instead of the "official" location?

Reply:
The reason was that there is no official tag for downloading version 1.8.0 and I was unable to find the right way to download a snapshot from master trunk.
The file should be this one:
https://sourceforge.net/code-snapshots/git/p/pr/projectlibre/code.git/projectlibre-code-858bc0d6f82fdea6c67f145c768e70b48beb1b16.zip
Any help is appreciated.